Core.DomainModel/KendoConfig/OverviewType.cs

```diff
@@ -2,6 +2,7 @@
 {
     public enum OverviewType
     {
-        ItSystemUsage = 0
+        ItSystemUsage = 0,
+        ItContract = 1,
```

Comment:
When we extende the backend I think we should change the existing test to be a theory to test all members of the enum